mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -79,14 +79,15 @@ - (id)init {
                                                          forEventClass:kInternetEventClass
                                                             andEventID:kAEGetURL];
         
-        CFMachPortRef lowLevelEventTap = CGEventTapCreate(kCGAnnotatedSessionEventTap, kCGHeadInsertEventTap, kCGEventTapOptionListenOnly, NSFlagsChangedMask | NSLeftMouseDown | NSLeftMouseUp, (CGEventTapCallBack)eventTapFunction, nil);
+        self.lowLevelEventTap = CGEventTapCreate(kCGAnnotatedSessionEventTap, kCGHeadInsertEventTap, kCGEventTapOptionListenOnly, NSFlagsChangedMask | NSLeftMouseDown | NSLeftMouseUp, (CGEventTapCallBack)eventTapFunction, nil);
         
-        if (!lowLevelEventTap)
+        if (!self.lowLevelEventTap) {
             NSLog(@"Can't tap into low level events!");
+        }
         else
-            CFRelease(lowLevelEventTap);
+            CFRelease(self.lowLevelEventTap);
```

Comment:
We prefer to use open and close curly braces even for single-statement blocks. I know a lot of the existing code doesn't do this but let's tidy as we go.